docs/cli-reference.md

```diff
@@ -0,0 +1,90 @@
+# StepCI CLI reference
+The following documents evety command and flag avaliable for use in StepCI's command-line interface.
+
+With the CLI you can:
+- Run workflows
+
+Follow the [getting started guide](https://github.com/stepci/stepci/blob/main/docs/get-started.md) for setup instructions
```

Comment:
Change the link to (../get-started.md). Also change setup instructions > installation instructions